src/app/component/create-loan/create-loan-form.tsx

```diff
@@ -1,110 +1,156 @@
-                    <input type="number" required  className='p-2 m-2 bg-slate-200 rounded-md'/>
-                    <select name="" id="" className='bg-slate-100 h-10'>
-                        <option value="ETH">USD</option>
-                        <option value="USDC">USDC</option>
-                        <option value="LINK">LINK</option>
-                        <option value="USD">ETH</option>
-                    </select> 
-                    </div>
-                </div>
-                <div>
-                    <label htmlFor="">Loan period</label>
+  const handleGetBlockTime = (s: string) => {
+    const n = parseInt(s);
+    if (isLoading) {
+      return <div>Fetching block number</div>;
+    }
+    if (isError) {
+      return <div>Error Fetching block number</div>;
+    }
+    let amountAdd = 216000 * n; //216000 is the number of blocks that represents 30 days
+    let loanRepayDeadLineAddedAmount = BigInt(amountAdd);
+    let loanRepayDeadLine = loanRepayDeadLineAddedAmount;
+    if (data?.toString().length) {
+      loanRepayDeadLine = loanRepayDeadLine + data;
+      return <div> the date in blocks is {loanRepayDeadLine.toString()}</div>;
+    }
 
-                    <div className='bg-slate-100 rounded-xl flex'>                   
-                    <span className='p-2 m-2 bg-slate-200 rounded-md w-60'>Payment in months</span>
-                    <select className='bg-slate-100 h-10' value={selectedValue} onChange={handleSelectChange}>
-                        {[1, 2, 3, 4, 5, 6,7,8,9,10,11,12].map((value) => (
-                        <option key={value} value={value.toString()}>
-                            {value}
-                        </option> 
-                        ))}
-                    </select>
-                    </div>
+    return <div>none</div>;
+  };
 
-                </div>
-            </form>
-           
+  const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedValue(event.target.value);
+  };
+
+  return (
+    <>
+      <section>
+        <div className="w-full  mx-auto">
+          <h3 className="text-textBlack font-semibold leading-[23.4px] my-2">
+            Create Loan position
+          </h3>
+          <form action="">
+            <div>
+              <label htmlFor="">Request Amount</label>
+              <div className="bg-slate-100 rounded-xl">
+                <input
+                  type="number"
+                  required
+                  placeholder="Enter Amount"
+                  className="p-2 m-2 bg-slate-200 outline-0 rounded-md"
+                />
+                <select
+                  name=""
+                  id=""
+                  className="bg-slate-100 h-10 cursor-pointer "
+                >
+                  <option value="ETH">USD</option>
+                  <option value="USDC">USDC</option>
+                  <option value="LINK">LINK</option>
+                  <option value="USD">ETH</option>
```

Comment:
This needs to be from a config object that we can save in `config.ts` or `conts.ts` 
```
ALLOWED_COINS = [{ value: "ETH", icon: "...", token: "ETH" }]
```

Reply:
After that make this select a input common component <SelectToken  value={} onChange={}/>

Reply:
resolved